frontend/src/utils/getDateAsLocalizedString.tsx

```diff
@@ -0,0 +1,34 @@
+import { getLocalizedDayjs } from '@mtes-mct/monitor-ui'
+
+export function formatDateLabel(dateLabel: string) {
```

Comment:
I don't really understand what this function is doing, the day is uppercased ? Maybe we can rename the function like this ?